css/build.css

```diff
@@ -19,7 +19,10 @@
     -webkit-font-smoothing: antialiased;
 }
 
-img[data-image-id] {
+img[data-image-id],
+canvas.lazy-placeholder {
     max-width: 100%;
     height: auto;
+    /* Uncertain at the moment where the 5px comes from at the moment, but is consistent across devices and image size */
+    margin-bottom: -5px;
```

Comment:
I'm guessing it's related, because the position of the wrapping `<span>` around the `<img>` and `<canvas>` are different. But couldn't find a fix to prove that is the cause.